src/vs/base/node/id.ts

```diff
@@ -99,3 +100,16 @@ async function getMacMachineId(errorLogger: (error: any) => void): Promise<strin
 		return undefined;
 	}
 }
+
+const SQM_KEY: string = '\\Software\\Microsoft\\SQMClient';
```

Comment:
Can we write a unit test for this method?

Reply:
Done